prow/prstatus/prstatus.go

```diff
@@ -35,11 +35,12 @@ import (
 )
 
 const (
-	loginSession   = "github_login"
-	githubEndpoint = "https://api.github.com"
-	tokenSession   = "access-token-session"
-	tokenKey       = "access-token"
-	loginKey       = "login"
+	loginSession    = "github_login"
+	githubEndpoint  = "https://api.github.com"
```

Comment:
Can we expose these constants in the GitHub package and use them where appropriate? Would hate to typo `graphql` in a flag default...

Reply:
I'm fine with it, hopefully there won't be any cyclical dependency issues. :crossed_fingers:

Reply:
This also kinda touches on the shared flagset conversation we had a week or so ago. Do we want to throw these defaults (constants) in `prow/github` and inherit them in `prow/flagutils/github.go` and in these desperate applications? Or do we want to leave these separate?